### ISSUE TITLE
oauth2 token only works once

### ISSUE THREAD

**Issue by andrew521** [jborchma/qtrade#66]
I believe QT made some changes in the last year and things might have been broken. So, these are my steps:
- I go at https://apphub.questrade.com/UI/UserApps.aspx, delete old token and create a new one
- qtrade = Questrade(access_code=token) runs successfully once. Second time throws  Error: 400 Client Error: Bad Request for url: https://login.questrade.com/oauth2/token?grant_type=refresh_token&refresh_token=xxxxxxxxxxxxxxxxxxxx

Basically, I have to create a new token for each run.

Am I doing something wrong?

**Comment by jborchma** [comment]
So I think you should be using the `refresh_access_token`-method to get a refreshed token. The API-docs for the method can be found [here](https://jborchma.github.io/qtrade/api/qtrade.html#qtrade.questrade.Questrade.refresh_access_token).

**Comment by jborchma** [comment]
Closing this as I assume my reply solved the issue.
